sdk/core/Microsoft.Extensions.Azure/tests/TestClientOptionsMultipleParameters.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Pipeline;
+
+namespace Azure.Core.Extensions.Tests
+{
+    internal class TestClientOptionsMultipleParameters : ClientOptions
+    {
+        public string OtherParameter { get; }
+        public ServiceVersion Version { get; }
+
+        public enum ServiceVersion
+        {
+            A,
+            B,
+            C,
+            D
+        }
+        public TestClientOptionsMultipleParameters(string otherParameter = "some default value", ServiceVersion version = ServiceVersion.D)
```

Comment:
Nit - ServiceVersion always has to be the first parameter per the guidelines

Reply:
People don't follow the guideline :) I'd rather be safe.